<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssemblyRouteCardClass</class>
 <widget class="QWidget" name="AssemblyRouteCardClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AssemblyRouteCard</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Xlsx</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Script Path:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="scriptPath"/>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="loadScript">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Script:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QPushButton" name="run">
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>CAD</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Input DWG:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="InputPath"/>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="loadInput">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" rowspan="2">
          <widget class="QToolButton" name="export_2">
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Output Xlsx:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="OutputPath"/>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="LoadOutput">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>551</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="log"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AssemblyRouteCard.qrc"/>
 </resources>
 <connections/>
</ui>
